model/Extension/Classes/Extension.md

```diff
@@ -29,3 +29,25 @@ This approach serves multiple purposes:
 
 - name: Extension
 - Instantiability: Abstract
+
+## Summary  @zh-Hans
+
+`Element`某个方面的特征描述，以广义的方式与`Element`关联。
+
+## Description  @zh-Hans
+
+`Extension`是对某个`Element`某个方面的特征描述，以广义的方式与该`Element`关联。
+
+`Extension`并不是通过特定目的的对象属性与某个特定`Element`关联，而是通过一个共同的广义对象属性与它所描述的`Element`关联。
+
+此方法有多种用途：
+1. 支持基于配置文件的`Element`扩展特征描述。允许在任何SPDX配置文件和命名空间内指定和表达`Element`特征描述扩展，无需对其他配置文件或命名空间进行更改，也无需对远程类进行本地子类化（在某些情况下可能会阻碍生态系统的互操作性）。
```

Comment:
```suggestion
此方法有多种用途：

1. 支持基于配置文件的`Element`扩展特征描述。允许在任何SPDX配置文件和命名空间内指定和表达`Element`特征描述扩展，无需对其他配置文件或命名空间进行更改，也无需对远程类进行本地子类化（在某些情况下可能会阻碍生态系统的互操作性）。
```

Please put one blank line before and after the numbered/bullet list. MkDocs can be picky about this. Thank you.